source/includes/maintenance-mode/_70-info.md.erb

```diff
@@ -24,21 +23,11 @@ ETag: "cbc5f2d5b9c13a2cc1b1efb3d8a6155d"
       "href" : "https://api.gocd.org/current/#maintenance-mode-info"
     }
   },
-  "_embedded" : {
-    "is_maintenance_mode" : true,
-    "metadata" : {
-      "updated_by" : "admin",
-      "updated_on" : "2019-01-02T04:18:28Z"
-    },
-    "attributes" : {
```

Comment:
This node is present when the maintainance mode is turned on, can you check at your end?

Reply:
With maintenance mode disabled:
```json
{
  "_links" : {
    "self" : {
      "href" : "http://127.0.0.1:8153/go/api/admin/maintenance_mode/info"
    },
    "doc" : {
      "href" : "https://api.gocd.org/19.5.0/#maintenance-mode-info"
    }
  },
  "is_maintenance_mode" : false,
  "metadata" : {
    "updated_by" : "admin",
    "updated_on" : "2019-07-16T10:55:14Z"
  }
}
```

With maintenance mode enabled:
```json
{
  "_links" : {
    "self" : {
      "href" : "http://127.0.0.1:8153/go/api/admin/maintenance_mode/info"
    },
    "doc" : {
      "href" : "https://api.gocd.org/19.5.0/#maintenance-mode-info"
    }
  },
  "is_maintenance_mode" : true,
  "metadata" : {
    "updated_by" : "admin",
    "updated_on" : "2019-07-16T15:17:48Z"
  },
  "attributes" : {
    "has_running_systems" : false,
    "running_systems" : {
      "material_update_in_progress" : [ ],
      "building_jobs" : [ ],
      "scheduled_jobs" : [ ]
    }
  }
}
```

I can update the  PR to contain 
- output for `"is_maintenance_mode" : true` or
- both cases?